src/index.html

```diff
@@ -3,11 +3,51 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div class="card-product" data-qa="card">
+      <img
+        class="card-product__image"
+        src="images/imac.jpeg"
+        alt="image of APPLE A1419"
+      >
+      <h1 class="card-product__title">
```

Comment:
```suggestion
      <h3 class="card-product__title">
```
The main header should be single on the page, but it's expected to have a lot of cards on the page

Reply:
Tag has been changed